src/utils/fetchClient.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the eslint-disable comment and fixing the underlying issue. Disabling eslint rules can lead to code quality issues. 🚨